environments/dev/backend_lb_config.yaml

```
@@ -0,0 +1,11 @@
+---
+
+gateways:
+    - gateway_configuration:
+        host_name_suffix: service.core-compute-dev.internal
+        ssl_host_name_suffix: dev.platform.hmcts.net
+        certificate_name: wildcard-dev-platform-hmcts-net
```

this certificate likely needs ordering 
https://hmcts.github.io/ways-of-working/path-to-live/tls-certificates.html#tls-certificates
(the vault will be cftapps-dev not prod)

```suggestion
        certificate_name: wildcard-preview-platform-hmcts-net
```